submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/common/BaseIntegrationTest.kt

```diff
@@ -16,4 +25,30 @@ internal open class BaseIntegrationTest(private val tempFolder: TemporaryFolder)
         System.setProperty("app.tempDirPath", temp.absolutePath)
         System.setProperty("app.security.filesDirPath", dropbox.absolutePath)
     }
+
+    protected fun getWebClient(serverPort: Int, user: TestUser): BioWebClient {
+        val securityClient = SecurityWebClient.create("http://localhost:$serverPort")
+        securityClient.registerUser(user.asRegisterRequest())
+
+        return securityClient.getAuthenticatedClient(user.email, user.password)
+    }
+
+    protected fun submitString(
+        webClient: BioWebClient,
+        submission: String,
+        format: SubmissionFormat = TSV,
+        files: List<File> = emptyList()
+    ) = assertSuccessfulResponse(webClient.submitSingle(submission, format, files))
+
+    protected fun submitFile(webClient: BioWebClient, submission: File, files: List<File> = emptyList()) =
```

Comment:
this method does help to reuse code but actually make code harder to understand and maintain as user will need to go to this method definition to check how submission is submitted.  `webClient.submitSingle(submission, files)` is IMO way more clear even is duplicated code, as commented before IMO our final goal is no make maintainable code not to avoid duplication (if we can do both much better).

Reply:
Done